[2024-09-25T11:10:35.267+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-25T11:10:35.408+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.extract_transform.orders_history manual__2024-09-25T11:07:53.721667+00:00 [queued]>
[2024-09-25T11:10:35.465+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.pipeline.warehouse.step_1.extract_transform.orders_history manual__2024-09-25T11:07:53.721667+00:00 [queued]>
[2024-09-25T11:10:35.467+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-25T11:10:35.534+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): pipeline.warehouse.step_1.extract_transform.orders_history> on 2024-09-25 11:07:53.721667+00:00
[2024-09-25T11:10:35.567+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=825) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-25T11:10:35.576+0000] {standard_task_runner.py:72} INFO - Started process 842 to run task
[2024-09-25T11:10:35.615+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'pipeline.warehouse.step_1.extract_transform.orders_history', 'manual__2024-09-25T11:07:53.721667+00:00', '--job-id', '210', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline/run.py', '--cfg-path', '/tmp/tmpln_ay25w']
[2024-09-25T11:10:35.731+0000] {standard_task_runner.py:105} INFO - Job 210: Subtask pipeline.warehouse.step_1.extract_transform.orders_history
[2024-09-25T11:10:36.504+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline.pipeline.warehouse.step_1.extract_transform.orders_history manual__2024-09-25T11:07:53.721667+00:00 [running]> on host e10075799413
[2024-09-25T11:10:38.404+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='pipeline.warehouse.step_1.extract_transform.orders_history' AIRFLOW_CTX_EXECUTION_DATE='2024-09-25T11:07:53.721667+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-25T11:07:53.721667+00:00'
[2024-09-25T11:10:38.422+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-25T11:10:38.870+0000] {base.py:84} INFO - Retrieving connection 'staging_db'
[2024-09-25T11:10:42.315+0000] {base.py:84} INFO - Retrieving connection 'warehouse_db'
[2024-09-25T11:10:47.198+0000] {base.py:84} INFO - Retrieving connection 'minio'
[2024-09-25T11:10:47.310+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-25T11:10:47.344+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-25T11:10:47.345+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=pipeline.warehouse.step_1.extract_transform.orders_history, run_id=manual__2024-09-25T11:07:53.721667+00:00, execution_date=20240925T110753, start_date=20240925T111035, end_date=20240925T111047
[2024-09-25T11:10:47.445+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-25T11:10:47.829+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-25T11:10:47.844+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
